README.md

```diff
@@ -1,9 +1,9 @@
-# djinni-documentation
-
-**[:arrow_right: Documentation Website :arrow_left:](https://cross-language-cpp.github.io/djinni-documentation/)**
+# Djinni Documentation
 
 Documentation for the **cross-language-cpp** fork of djinni.
 
+:arrow_right: Available at: **[djinni.xlcpp.dev](https://djinni.xlcpp.dev/)**
```

Comment:
Pls tell me when the `djinni.dev` domain is set up. Then I'll have to update this again :)

Reply:
sure, but you know now that this will take at least some/many hours :-)